keyboards/dreamystate/dy16/config.h

```diff
@@ -0,0 +1,210 @@
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+*/
+#define MATRIX_ROW_PINS { B7, D0, D1, D5, C7 }
+#define MATRIX_COL_PINS { F7, F6, F5, F4 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+/* Split Keyboard specific options, make sure you have 'SPLIT_KEYBOARD = yes' in your rules.mk, and define SOFT_SERIAL_PIN.
+ */
+// #define SOFT_SERIAL_PIN D0 // or D1, D2, D3, E6
+
+#define RGB_DI_PIN E6
+// #ifdef RGB_DI_PIN
+  #define RGBLED_NUM 16
+//   #define RGBLIGHT_HUE_STEP 8
+//   #define RGBLIGHT_SAT_STEP 8
+//   #define RGBLIGHT_VAL_STEP 8
+  #define RGBLIGHT_LIMIT_VAL 240 /* The maximum brightness level */
+  #define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
+// /*== all animations enable ==*/
+  #define RGBLIGHT_ANIMATIONS
+// /*== or choose animations ==*/
+//   #define RGBLIGHT_EFFECT_BREATHING
+//   #define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//   #define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//   #define RGBLIGHT_EFFECT_SNAKE
+//   #define RGBLIGHT_EFFECT_KNIGHT
+//   #define RGBLIGHT_EFFECT_CHRISTMAS
+//   #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//   #define RGBLIGHT_EFFECT_RGB_TEST
+//   #define RGBLIGHT_EFFECT_ALTERNATING
+// #endif
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+/* Audio output
+ * 
+ * STARTUP_SONG // plays when the keyboard starts up (audio.c)
+ * GOODBYE_SONG // plays when you press the RESET key (quantum.c)
+ * AG_NORM_SONG // plays when you press AG_NORM (quantum.c)
+ * AG_SWAP_SONG // plays when you press AG_SWAP (quantum.c)
+ * CG_NORM_SONG // plays when you press CG_NORM (quantum.c)
+ * CG_SWAP_SONG // plays when you press CG_SWAP (quantum.c)
+ * MUSIC_ON_SONG // plays when music mode is activated (process_music.c)
+ * MUSIC_OFF_SONG // plays when music mode is deactivated (process_music.c)
+ * CHROMATIC_SONG // plays when the chromatic music mode is selected (process_music.c)
+ * GUITAR_SONG // plays when the guitar music mode is selected (process_music.c)
+ * VIOLIN_SONG // plays when the violin music mode is selected (process_music.c)
+ * MAJOR_SONG // plays when the major music mode is selected (process_music.c)
+*/
+#ifdef AUDIO_ENABLE
+  #define AUDIO_VOICES
+  #define C6_AUDIO
+  #define STARTUP_SONG SONG(STARTUP_SOUND)
+#endif
+
+/* Encoder output*/
+#define ENCODERS_PAD_A { D4,B6 }
+#define ENCODERS_PAD_B { D6,C7 }
+#define ENCODER_RESOLUTION 4
+
+/* define if matrix has ghost (lacks anti-ghosting diodes) */
+//#define MATRIX_HAS_GHOST
+
+/* number of backlight levels */
+// #define BACKLIGHT_PIN B6
+// #define BACKLIGHT_BREATHING
+// #define BACKLIGHT_LEVELS 5
+
```

Comment:
Don't need this as the board doesn't support Backlight (single-color non-per-key).

```suggestion
```